frontend/src/utils/localForage.ts

```diff
@@ -3,6 +3,7 @@ import * as localForage from 'localforage';
 export const localForageKeys = {
   API_KEY: 'API_KEY',
   USER: 'USER',
+  EVENT_ID: 'EVENT_ID',
```

Comment:
This is leading to all kinds of weird errors ... you only want to add this to local storage that you need to persist after refresh, and they have to definitely carry some decent significance across multiple pages ... this doesn't qualify. For one, it results in a weird bug where when I reload the page, there's still a run instance showing without a selected event

![image](https://user-images.githubusercontent.com/5672438/83998801-114b5a80-a96a-11ea-9612-18d236694174.png)